packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2417,13 +2417,20 @@ is required to delete the user.
 | id        | `id` of user                                                                          | URL        | String   | TRUE     |         |
 | sendEmail | Sends a deactivation email to the administrator if `true`.  Default value is `false`. | Query      | Boolean  | FALSE    | FALSE   |
 
+> User deletion can also be performed asynchronously. To invoke asynchronous user deletion, pass an HTTP header
+> `Prefer: respond-async` with the request. This header is also supported by user deactivation, which will
```

Comment:
Change:
This header is also supported by user deactivation, which will be performed.....

To:
This header is also supported by user deactivation, which is performed asynchronously if the delete endpoint is invoked on a user that hasn't been deactivated.

Reply:
Updated.

Reply:
This is technically incorrect now.  It is only performed asynchronously if that condition holds, and this header is also supplied.

Perhaps `This header is also supported by user deactivation, which is performed if the delete endpoint is invoked on a user that hasn't been deactivated`?

Reply:
Updated to Dayton's suggestion.